vaadin-component-demo.html

```diff
@@ -145,9 +145,10 @@
 
       _configChanged() {
         if (this.configSrc) {
-          fetch(this.configSrc)
-            .then(res => res.json())
-            .then(json => {
+          var xhr = new XMLHttpRequest();
+          xhr.onreadystatechange = () => {
```

Comment:
`onload` is more readable, you won't need to check `readyState`, just `status` http://youmightnotneedjquery.com/